src/commands/memory-malloc-stats.json

```diff
@@ -1,7 +1,7 @@
 {
     "MALLOC-STATS": {
         "summary": "Show allocator internal stats",
-        "complexity": "PATCH__TBD__71__",
+        "complexity": "NA",
```

Comment:
it could be slow... but we really can't tell anything.

Reply:
since it's free text, let's mention that it could be slow, and depends among others, on how much memory is allocated.